spec/services/flight_api_service_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+require 'json'
+
+def app
+  FlightApiServiceController
+end
+
+describe 'I can connect and recieve a response from the flight API' do
+  it "I get JSON data back" do
+    params = {
+              :fly_from => 'DEN',
+              :date_from => '30/01/2021',
+              :date_to => '30/01/2021',
+              :nights_in_dst_from => 5,
+              :nights_in_dst_to => 5,
+              :flight_type => 'round',
+              :one_for_city => 1,
+              :partner_market => 'us',
+              :curr => 'USD',
+              :sort => 'price',
+              :ret_from_diff_airport => 0,
+              :partner => 'picky',
+              :limit => 20
+              }
+    get '/flights', params
+    expect(last_response.status).to eq(200)
```

Comment:
Rack::Test is used to test what is being returned by the API, e.g. last_response.status